### DISCUSSION TITLE
Does Remotely run in safe mode?

### DISCUSSION THREAD

**Discussion by stevesobol** [immense/Remotely#951] [Q&A]
Hello,

I have an antivirus uninstaller that refuses to run unless you're in safe mode, and I'm wondering if Remotely will run in safe mode. Thanks.

**Comment by bierco** [comment]
Hello,

I had the same problem with eset, remotely does not run in safe mode but you can create an uninstall script by sending it from Remotely in normal mode, if the antivirus you have has uninstall parameters better because when you send the script, it will run in safe mode and restart in normal mode, if you need help, if you need me avias me with eset I managed to do it on more than 2000 machines that changed the antivirus but remotely helped a lot.
